<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RANADEEP_BANIK" ignored="0" name="Test_Framework-Basic_Tests" tests="2" failures="1" timestamp="16 Apr 2020 18:39:23 GMT" time="33.352" errors="0">
  <testcase name="checkCaseCalculationsOfTripuraIsGreaterThan10" time="3.24" classname="com.ui.testautomation.CovidIndiaUITest">
    <skipped/>
  </testcase> <!-- checkCaseCalculationsOfTripuraIsGreaterThan10 -->
  <testcase name="checkCaseCalculationsOfTripuraIsGreaterThan10" time="3.651" classname="com.ui.testautomation.CovidIndiaUITest">
    <failure type="java.lang.AssertionError" message="Case count is not greater than 10 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Case count is not greater than 10 expected [true] but found [false]
	at com.app.testautomation.utilities.Assertions.assertTrue(Assertions.java:24)
	at com.ui.testautomation.CovidIndiaUITest.checkCaseCalculationsOfTripuraIsGreaterThan10(CovidIndiaUITest.java:55)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkCaseCalculationsOfTripuraIsGreaterThan10 -->
</testsuite> <!-- Test_Framework-Basic_Tests -->
